.github/workflows/vulndb.yaml

```diff
@@ -40,12 +44,37 @@ jobs:
       uses: actions/setup-go@v5
       with:
         go-version: 1.22
+
     - name: Build the database (this takes some time)
       run: |
         go run ./cmd/devguard-cli/main.go vulndb repair --startIndex=0
```

Comment:
I think we can improve this in the future. It is not necessary to start with a fresh instance. It might be more suitable to first call the import command to have the database from "yesterday". After that we can call repair without the startIndex flag, to sync the missing day.